napalm/eos.py

```diff
@@ -484,3 +484,18 @@ def get_lldp_neighbors_detail(self, interface = ''):
                 )
 
         return lldp_neighbors_out
+
+    def cli(self, command = None):
+
+        if type(command) is not str:
+            return 'Please enter a valid command!'
+
+        commands = list()
+        commands.append(command)
+
+        try:
+            return self.device.run_commands(commands, encoding = 'text')[0].get('output')
+        except pyeapi.eapilib.CommandError:
+            return 'Unrecognized command: "{0}"'.format(command)
```

Comment:
Could we raise an exception instead? I think it's better if we raise an Exception so you can handle them (or ignore them) programatically. By looking at your code I guess we will need at least two:

1. CommandError
2. CommandTimeout

The messages are fine, it's just that I'd rather know that an error occurred without having to parse the return string.